lib/morpheus.gi

```diff
@@ -2818,6 +2818,44 @@ local d,iso,a,b,c,o,s,two,rt,r,z,e,y,re,m,gens,cnt,lim,p,
 end);
 
 
+BindGlobal("IsomorphismPGroups",function(G,H)
+local s,p,eG,eH,fG,fH,pc,imgs;
+  s:=Size(G);
+  if Size(H)<>s then return fail;fi;
+  p:=Collected(Factors(s));
+  if Length(p)>1 then TryNextMethod();fi;
+  p:=p[1][1];
+  eG:=CallFuncList(ValueGlobal("EpimorphismPqStandardPresentation"),[G]:
+    Prime:=p);
+  eH:=CallFuncList(ValueGlobal("EpimorphismPqStandardPresentation"),[H]:
```

Comment:
Seems this function is not actually tested by the test suite, at least in CI -- there very well may be an existing test triggering this function, but of course only if `anupq` can be loaded... but I think we don't actually compile `anupq` when running the CI tests. I guess we could change that by adding `anupq` to line 44 in `dev/ci-download-pkgs.sh`

Reply:
Even with ANUPQ this can be quickly quite costly and thus did not seem a good idea to add. I'm happy to provide a test if it is requested.